.travis.yml

```diff
@@ -34,12 +34,24 @@ jobs:
       env:
         - ALPAKKA_KAFKA_SNAPSHOT="-Dbuild.alpakka.kafka.version=2.0.3+7-edfcbb02" # Remove when Alpakka Kafka 2.0.4 released
         - CMD="test"
-      name: "Run tests with Scala 2.13 and AdoptOpenJDK 8"
+      name: "Run tests with Scala 2.12 and AdoptOpenJDK 8"
     - env:
         - ALPAKKA_KAFKA_SNAPSHOT="-Dbuild.alpakka.kafka.version=2.0.3+7-edfcbb02" # Remove when Alpakka Kafka 2.0.4 released
         - JDK="adopt@~1.11-0"
         - _JAVA_OPTIONS="-XX:+UnlockExperimentalVMOptions -XX:+UseJVMCICompiler"
         - CMD="test"
+      name: "Run tests with Scala 2.12 and AdoptOpenJDK 11"
+      
+    - env:
+        - ALPAKKA_KAFKA_SNAPSHOT="-Dbuild.alpakka.kafka.version=2.0.3+7-edfcbb02" # Remove when Alpakka Kafka 2.0.4 released
+        - CMD="++2.13.3 test"
```

Comment:
In akka/akka we use the Travis scala: https://github.com/akka/akka/blob/master/.travis.yml#L5
What is best?

On the other hand we could also consider to test with one of them and compile with both if we want to reduce PR validation time. I think it's very rare to find any runtime problems with a different Scala version.

Reply:
The goal was to avoid the usage of Travis matrix and make it clear on each stage what is being used. We have been doing like that in a few projects (not Akka core indeed).